### PR TITLE
Updates and reorganization of the influxd-ctl and influx utilities page

### PR DESCRIPTION
**Pull request by stevebang**
This effort included renaming, reformatting, and reorganizing the cluster management utilities, including `influxd-ctl` and `influx` (well, a pointer to the docs on that). 

* In the existing content, the `influxd-ctl` commands where organized under the "Arguments" section -- that made no sense and are now under the "Commands" [duh] section.
* The example descriptions have been moved to above the code, to give context to the examples.
* Split out the syntax for each command to subsections labelled "Syntax".
* And much, much more.
* this "Cluster commands" page doesn't really seem that it should be under "Features" -- moved in this PR to the "Administration" section.  And, thinking about renaming the page and menu titles...something like "Cluster management utilities"(?)...still thinking...share your thoughts!

Open to more ideas for improvements (in this or later PRs)!

### CODE REVIEW

File: content/enterprise_influxdb/v1.5/features/cluster-commands.md [hunk 1]
```diff
@@ -9,14 +10,21 @@ menu:
...
+InfluxDB Enterprise includes two utilities for interacting with and managing your clusters. The [`influxd-ctl`](#influxd-ctl) utility provides commands for managing your InfluxDB Enterprise clusters. The [`influx` command line interface](#influx) is used for interacting with and managing your data.
 
 #### Content
 
-* [influxd-ctl](#influxd-ctl)
+* [`influxd-ctl` cluster management utility](#influxd-ctl)
     * [Syntax](#syntax)
-    * [Global options](#global-options)
-    * [Arguments](#arguments)
+    * [Global options](#global-arguments)
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.5/features/cluster-commands.md]: Change `#global-arguments` to `#global-options`

File: content/enterprise_influxdb/v1.5/features/cluster-commands.md [hunk 2]
```diff
@@ -34,199 +42,237 @@ Use the cluster command line tools [`influxd-ctl`](#influxd-ctl) and [`influx`](
         * [`update-data`](#update-data)
         * [`token`](#token)
         * [`truncate-shards`](#truncate-shards)
-* [influx](#influx)
+* [`influx` command line interface](#influx-cli-utility)
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.5/features/cluster-commands.md]: Change `#influx-cli-utility` to `#influx-command-line-interface-cli`

File: content/enterprise_influxdb/v1.5/features/cluster-commands.md [hunk 3]
```diff
@@ -9,14 +10,21 @@ menu:
     parent: Enterprise features
 ---
 
-Use the cluster command line tools [`influxd-ctl`](#influxd-ctl) and [`influx`](#influx) to interact with your cluster and data.
+InfluxDB Enterprise includes two utilities for interacting with and managing your clusters. The [`influxd-ctl`](#influxd-ctl) utility provides commands for managing your InfluxDB Enterprise clusters. The [`influx` command line interface](#influx) is used for interacting with and managing your data.
 
 #### Content
 
-* [influxd-ctl](#influxd-ctl)
+* [`influxd-ctl` cluster management utility](#influxd-ctl)
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.5/features/cluster-commands.md]: Change `#influxd-ctl` to `#influxd-ctl-cluster-management-utility`.

File: content/enterprise_influxdb/v1.5/features/cluster-commands.md [hunk 4]
```diff
@@ -34,199 +42,237 @@ Use the cluster command line tools [`influxd-ctl`](#influxd-ctl) and [`influx`](
...
-* [influx](#influx)
+* [`influx` command line interface](#influx-cli-utility)
 
 
-## `influxd-ctl`
+## `influxd-ctl` cluster management utility
 
-The `influxd-ctl` tool is available on all [meta nodes](/enterprise_influxdb/v1.5/concepts/glossary/#meta-node).
-Use `influxd-ctl` to manage your cluster nodes, backup data, restore data, and rebalance your cluster.
+Use the `influxd-ctl` cluster management utility to manage your cluster nodes, back up and restore data, and rebalance your cluster.
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.5/features/cluster-commands.md]: I think "cluster" in "manage your cluster nodes" should be possessive:

> manage your cluster's nodes,  ...


Or leave out "your cluster" so it reads:

> Use the `influxd-ctl` cluster management utility to manage nodes, back up and restore data, and rebalance your cluster.

File: content/enterprise_influxdb/v1.5/features/cluster-commands.md [hunk 5]
```diff
@@ -34,199 +42,237 @@ Use the cluster command line tools [`influxd-ctl`](#influxd-ctl) and [`influx`](
...
 ```
-influxd-ctl [global-options] <command> [arguments]
+influxd-ctl [ global-options ] <command> [ arguments ]
 ```
 
 ### Global options
 
-`-auth-type [ none | basic | jwt ]`  
-Specify the type of authentication to use. The default authentication type is `none`.
+Global options for the `influxd-ctl` utility are used to manage authentication, bindings, .
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.5/features/cluster-commands.md]: Remove the command and space at the end of the sentence.

File: content/enterprise_influxdb/v1.5/features/cluster-commands.md [hunk 6]
```diff
@@ -34,199 +42,237 @@ Use the cluster command line tools [`influxd-ctl`](#influxd-ctl) and [`influx`](
...
+Specify the JSON Web Token (JWT) shared secret. This argument is ignored if `-auth-type jwt` isn't specified.
 
-#### Example 1: Bind to a remote meta node
+#### `[ -user <username> ]`  
+
+Specify the user’s username. This argument is ignored if `-auth-type basic` isn’t specified.
+
+### Examples
+
+The following examples use the `influxd-ctl` utility [`show` option](#show).
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.5/features/cluster-commands.md]: "utility" should be possessive: "utility's"

File: content/enterprise_influxdb/v1.5/features/cluster-commands.md [hunk 7]
```diff
@@ -270,39 +315,48 @@ Backed up to backup_dir in 51.388233ms, transferred 333793 bytes
...
-
 #### `copy-shard-status`
 
-Shows all in-progress [copy shard](#copy-shard) operations, including the shard's source node, destination node, database, [retention policy](/influxdb/v1.5/concepts/glossary/#retention-policy-rp), shard id, total size, current size, and the operation's start time.
+Shows all in-progress [copy shard](#copy-shard) operations, including the shard's source node, destination node, database, [retention policy](/influxdb/v1.5/concepts/glossary/#retention-policy-rp), shard ID, total size, current size, and the operation's start time.
+
+##### Syntax
+
 ```
 copy-shard-status
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.5/features/cluster-commands.md]: I think the syntax examples should include the full command to provide the proper context.

```
influxd-ctl copy-shard-status
```

File: content/enterprise_influxdb/v1.5/features/cluster-commands.md [hunk 8]
```diff
@@ -311,34 +365,38 @@ Source                     Dest                       Database  Policy   ShardID
...
 
 Joins a meta node and/or data node to a cluster.
 By default, `influxd-ctl` joins the local meta node and/or data node into a new cluster.
 Use `join` instead of the [`add-meta`](#add-meta) or [`add-data`](#add-data) arguments when performing a [QuickStart Installation](/enterprise_influxdb/v1.5/quickstart_installation/cluster_installation/) of an InfluxEnterprise cluster.
+
+##### Syntax
+
 ```
-join [-v] [<meta-node-HTTP-bind-address>]
+join [-v] [ <meta-node-HTTP-bind-address> ]
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.5/features/cluster-commands.md]: Include `influxd-ctl` in the syntax example to provide proper context.

```
influxd-ctl join [-v] [ <meta-node-HTTP-bind-address ]
```

Also, is the `<meta-node-HTTP-bind-address>` optional? I think it's required.

File: content/enterprise_influxdb/v1.5/features/cluster-commands.md [hunk 9]
```diff
@@ -414,58 +474,66 @@ No data node found on data-node-03:8091!
...
-
 #### `kill-copy-shard`
 
 Aborts an in-progress [`copy-shard`](#copy-shard) command.
+
+##### Syntax
+
 ```
-kill-copy-shard <data-node-source-TCP-address> <data-node-destination-TCP-address> <shard-id>
+kill-copy-shard <data-node-source-TCP-address> <data-node-destination-TCP-address> <shard-ID>
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.5/features/cluster-commands.md]: Include `influxd-ctl` in the syntax example to provide proper context.

File: content/enterprise_influxdb/v1.5/features/cluster-commands.md [hunk 10]
```diff
@@ -414,58 +474,66 @@ No data node found on data-node-03:8091!
...
 
 <dt>The `leave` argument is destructive; it erases all metastore information from meta nodes and all data from data nodes.
-Use `leave` only if you want to *permanently* remove a node from a cluster.</dt>
+Use `leave` only if you want to *permanently* remove a node from a cluster.
+</dt>
+
+##### Syntax
 
 ```
 leave [-y]
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.5/features/cluster-commands.md]: Include `influxd-ctl` in the syntax example to provide proper context.

File: content/enterprise_influxdb/v1.5/features/cluster-commands.md [hunk 11]
```diff
@@ -519,162 +587,193 @@ Successfully left cluster
...
 <dt>The `remove-data` argument is destructive; it erases all data from the specified data node.
-Use `remove-data` only if you want to *permanently* remove a data node from a cluster.</dt>
+Use `remove-data` only if you want to *permanently* remove a data node from a cluster.
+</dt>
+
+##### Syntax
 
 ```
-remove-data [-force] <data-node-TCP-bind-address>
+remove-data [ -force ] <data-node-TCP-bind-address>
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.5/features/cluster-commands.md]: Include `influxd-ctl` in the syntax example to provide proper context.

File: content/enterprise_influxdb/v1.5/features/cluster-commands.md [hunk 12]
```diff
@@ -519,162 +587,193 @@ Successfully left cluster
...
 <dt>The `remove-meta` argument is destructive; it erases all metastore information from the specified meta node.
-Use `remove-meta` only if you want to *permanently* remove a meta node from a cluster.</dt>
+Use `remove-meta` only if you want to *permanently* remove a meta node from a cluster.
+</dt>
+
+##### Syntax
 
 ```
-remove-meta [-force | -tcpAddr <meta-node-TCP-bind_address> | -y] <meta-node-HTTP-bind-address>
+remove-meta [ -force | -tcpAddr <meta-node-TCP-bind_address> | -y ] <meta-node-HTTP-bind-address>
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.5/features/cluster-commands.md]: Include `influxd-ctl` in the syntax example to provide proper context.

File: content/enterprise_influxdb/v1.5/features/cluster-commands.md [hunk 13]
```diff
@@ -519,162 +587,193 @@ Successfully left cluster
...
-
 #### `remove-shard`
 
 Removes a shard from a data node.
 Removing a shard is an irrecoverable, destructive action; please be cautious with this command.
+
+##### Syntax
+
 ```
 remove-shard <data-node-source-TCP-address> <shard-id>
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.5/features/cluster-commands.md]: Include `influxd-ctl` in the syntax example to provide proper context.

File: content/enterprise_influxdb/v1.5/features/cluster-commands.md [hunk 14]
```diff
@@ -519,162 +587,193 @@ Successfully left cluster
...
 
 Restore a [backup](#backup) to an existing cluster or a new cluster.
 Note that he existing cluster must contain no data in the databases affected by the restore.
 Restore supports both full backups and incremental backups; the syntax for a restore differs depending on the backup type.
 
+##### Syntax
+
 ```
-restore [ -db <database> | -full | -list | -newdb <new-database> | -newrf <int> | -newrp <retention-policy> | -rp <retention policy | shard <shard-id> ] ( <path-to-backup-manifest-file> | <path-to-backup-directory> )
+restore [ -db <db_name> | -full | -list | -newdb <newdb_name> | -newrf <newrf_integer> | -newrp <newrp_name> | -rp <rp_name> | shard <shard_ID> ] ( <path-to-backup-manifest-file> | <path-to-backup-directory> )
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.5/features/cluster-commands.md]: Include `influxd-ctl` in the syntax example to provide proper context.

File: content/enterprise_influxdb/v1.5/features/cluster-commands.md [hunk 15]
```diff
@@ -519,162 +587,193 @@ Successfully left cluster
...
 
 Removed shard 31 from cluster-data-node-02:8088
 ```
 
-The command removes shard `31` from the data node running at `cluster-data-node-02:8088`.
-
 #### `restore`
 
 Restore a [backup](#backup) to an existing cluster or a new cluster.
 Note that he existing cluster must contain no data in the databases affected by the restore.
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.5/features/cluster-commands.md]: Change "he" to "the". This is important enough that I might pull it out into a blockquote or a <dt>.

File: content/enterprise_influxdb/v1.5/features/cluster-commands.md [hunk 16]
```diff
@@ -701,20 +800,23 @@ Copying data to <hostname>:8088... Copying data to <hostname>:8088... Done. Rest
...
 #### `show`
 
 Shows all [meta nodes](/enterprise_influxdb/v1.5/concepts/glossary/#meta-node) and [data nodes](/enterprise_influxdb/v1.5/concepts/glossary/#data-node) that are part of the cluster.
-The output includes the InfluxEnterprise version number.
+The output includes the InfluxDB Enterprise version number.
+
+##### Syntax
+
 ```
 show
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.5/features/cluster-commands.md]: Include `influxd-ctl` in the syntax example to provide proper context.

File: content/enterprise_influxdb/v1.5/features/cluster-commands.md [hunk 17]
```diff
@@ -733,20 +835,23 @@ cluster-node-02:8091	1.3.x-c1.3.x
...
-The output shows that the cluster includes three meta nodes and two data nodes.
-Every node is using InfluxEnterprise version `1.3.x-c1.3.x`.
-
 #### `show-shards`
 
-Shows the cluster's existing [shards](/influxdb/v1.5/concepts/glossary/#shard), including the shard's id, database, [retention policy](/influxdb/v1.5/concepts/glossary/#retention-policy-rp), desired number of copies, [shard group](/influxdb/v1.5/concepts/glossary/#shard-group), start time, end time, expiry time and [data node](/enterprise_influxdb/v1.5/concepts/glossary/#data-node) owners.
+Outputs details about existing [shards](/influxdb/v1.5/concepts/glossary/#shard) of the cluster, including shard ID, database, [retention policy](/influxdb/v1.5/concepts/glossary/#retention-policy-rp), desired replicas, [shard group](/influxdb/v1.5/concepts/glossary/#shard-group), starting timestamp, ending timestamp, expiration timestamp, and [data node](/enterprise_influxdb/v1.5/concepts/glossary/#data-node) owners.
+
 ```
 show-shards
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.5/features/cluster-commands.md]: Include `influxd-ctl` in the syntax example to provide proper context.

File: content/enterprise_influxdb/v1.5/features/cluster-commands.md [hunk 18]
```diff
@@ -758,91 +863,98 @@ ID  Database             Retention Policy  Desired Replicas  Shard Group  Start
...
-Finally, shard `51` appears on two data nodes: `cluster-data-node-01:8088` and `cluster-data-node-03:8088`.
-
 #### `update-data`
 
 Updates a data node's address in the [meta store](/enterprise_influxdb/v1.5/concepts/glossary/#meta-service).
+
+##### Syntax
+
 ```
 update-data <data-node-old-TCP-bind-address> <data-node-new-TCP-bind-address>
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.5/features/cluster-commands.md]: Include `influxd-ctl` in the syntax example to provide proper context.

File: content/enterprise_influxdb/v1.5/features/cluster-commands.md [hunk 19]
```diff
@@ -758,91 +863,98 @@ ID  Database             Retention Policy  Desired Replicas  Shard Group  Start
...
 #### `token`
 
 Generates a signed JSON Web Token (JWT) token.
-The token argument only works when using JWT authentication in the cluster and when using the [`-auth-type jwt`](#global-options) and [`-secret <shared-secret>`](#global-options) flags.
+The token argument only works when using JWT authentication in the cluster and when using the [`-auth-type jwt`](#global-arguments) and [`-secret <shared-secret>`](#global-arguments) arguments.
+
+##### Syntax
 
 ```
 token [-exp <duration>]
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.5/features/cluster-commands.md]: Include `influxd-ctl` in the syntax example to provide proper context.

File: content/enterprise_influxdb/v1.5/features/cluster-commands.md [hunk 20]
```diff
@@ -758,91 +863,98 @@ ID  Database             Retention Policy  Desired Replicas  Shard Group  Start
...
 #### `truncate-shards`
 
 Truncates hot [shards](/influxdb/v1.5/concepts/glossary/#shard), that is, shards that cover the time range that includes the current time ([`now()`](/influxdb/v1.5/concepts/glossary/#now)).
-`truncate-shards` creates a new shard and the system writes all new points to that shard.
+The `truncate-shards` command creates a new shard and the system writes all new points to that shard.
+
+##### Syntax
 
 ```
 truncate-shards [-delay <duration>]
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.5/features/cluster-commands.md]: Include `influxd-ctl` in the syntax example to provide proper context.

File: content/enterprise_influxdb/v1.5/administration/backup-and-restore.md [hunk 21]
```diff
@@ -171,19 +172,19 @@ Note that the existing cluster must contain no data in the affected databases.*
...
-influxd-ctl [global-options] restore [restore-options] <path-to-backup-directory>
+influxd-ctl [global-arguments] restore [restore-arguments] <path-to-backup-directory>
 ```
 
 \* The existing cluster can have data in the `_internal` database, the database
 that the system creates by default.
 The system automatically drops the `_internal` database when it performs a complete restore.
 
-#### Global options:
+#### Global arguments
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.5/administration/backup-and-restore.md]: I still like this should be options, but I'm not going to fight too hard on it 😄.

File: content/enterprise_influxdb/v1.5/administration/backup-and-restore.md [hunk 22]
```diff
@@ -171,19 +172,19 @@ Note that the existing cluster must contain no data in the affected databases.*
...
 
-Please see the [influxd-ctl documentation](/enterprise_influxdb/v1.5/features/cluster-commands/#global-options)
-for a complete list of the global `influxd-ctl` options.
+Please see the [influxd-ctl documentation](/enterprise_influxdb/v1.5/features/cluster-commands/#global-arguments)
+for a complete list of the global `influxd-ctl` arguments.
 
-#### Restore options:
+#### Restore arguments
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.5/administration/backup-and-restore.md]: Same as above.

File: content/enterprise_influxdb/v1.5/features/cluster-commands.md (hunk 3 above)
Comment (stevebang) [reply content/enterprise_influxdb/v1.5/features/cluster-commands.md]: done

File: content/enterprise_influxdb/v1.5/features/cluster-commands.md (hunk 1 above)
Comment (stevebang) [reply content/enterprise_influxdb/v1.5/features/cluster-commands.md]: done

File: content/enterprise_influxdb/v1.5/features/cluster-commands.md (hunk 2 above)
Comment (stevebang) [reply content/enterprise_influxdb/v1.5/features/cluster-commands.md]: done

File: content/enterprise_influxdb/v1.5/features/cluster-commands.md (hunk 4 above)
Comment (stevebang) [reply content/enterprise_influxdb/v1.5/features/cluster-commands.md]: It is correct to say "manage your cluster nodes" -- Oracle docs have lots of examples. I think possessive is overused too in our docs.

File: content/enterprise_influxdb/v1.5/features/cluster-commands.md (hunk 5 above)
Comment (stevebang) [reply content/enterprise_influxdb/v1.5/features/cluster-commands.md]: Fixed.

File: content/enterprise_influxdb/v1.5/features/cluster-commands.md (hunk 6 above)
Comment (stevebang) [reply content/enterprise_influxdb/v1.5/features/cluster-commands.md]: fixed

File: content/enterprise_influxdb/v1.5/features/cluster-commands.md (hunk 7 above)
Comment (stevebang) [reply content/enterprise_influxdb/v1.5/features/cluster-commands.md]: done

File: content/enterprise_influxdb/v1.5/features/cluster-commands.md (hunk 8 above)
Comment (stevebang) [reply content/enterprise_influxdb/v1.5/features/cluster-commands.md]: fixed

File: content/enterprise_influxdb/v1.5/features/cluster-commands.md (hunk 10 above)
Comment (stevebang) [reply content/enterprise_influxdb/v1.5/features/cluster-commands.md]: fixed

File: content/enterprise_influxdb/v1.5/features/cluster-commands.md (hunk 11 above)
Comment (stevebang) [reply content/enterprise_influxdb/v1.5/features/cluster-commands.md]: fixed

File: content/enterprise_influxdb/v1.5/features/cluster-commands.md (hunk 12 above)
Comment (stevebang) [reply content/enterprise_influxdb/v1.5/features/cluster-commands.md]: fixed

File: content/enterprise_influxdb/v1.5/features/cluster-commands.md (hunk 13 above)
Comment (stevebang) [reply content/enterprise_influxdb/v1.5/features/cluster-commands.md]: fixed

File: content/enterprise_influxdb/v1.5/features/cluster-commands.md (hunk 15 above)
Comment (stevebang) [reply content/enterprise_influxdb/v1.5/features/cluster-commands.md]: done

File: content/enterprise_influxdb/v1.5/features/cluster-commands.md (hunk 16 above)
Comment (stevebang) [reply content/enterprise_influxdb/v1.5/features/cluster-commands.md]: done

File: content/enterprise_influxdb/v1.5/features/cluster-commands.md (hunk 17 above)
Comment (stevebang) [reply content/enterprise_influxdb/v1.5/features/cluster-commands.md]: done

File: content/enterprise_influxdb/v1.5/features/cluster-commands.md (hunk 18 above)
Comment (stevebang) [reply content/enterprise_influxdb/v1.5/features/cluster-commands.md]: done

File: content/enterprise_influxdb/v1.5/features/cluster-commands.md (hunk 19 above)
Comment (stevebang) [reply content/enterprise_influxdb/v1.5/features/cluster-commands.md]: done

File: content/enterprise_influxdb/v1.5/features/cluster-commands.md (hunk 20 above)
Comment (stevebang) [reply content/enterprise_influxdb/v1.5/features/cluster-commands.md]: done

**Pull request merged**